packages/x-date-pickers/src/ClockPicker/ClockPicker.tsx

```diff
@@ -263,7 +263,10 @@ export const ClockPicker = React.forwardRef(function ClockPicker<TDate extends u
         return false;
       }
 
-      const validateTimeValue = (getRequestedTimePoint: (when: 'start' | 'end') => TDate) => {
+      const validateTimeValue = (
```

Comment:
I find this whole method with the pipe very convoluted
But I don't want to change anything non-trivial before knowing the codebase better